src/src/Commands/Tasks.cpp

```diff
@@ -67,7 +67,7 @@ String Command_Task_ClearAll(struct EventStruct *event, const char *Line)
   return return_command_success();
 }
 
-String Command_Task_EnableDisable(struct EventStruct *event, bool enable)
+String Command_Task_EnableDisable(struct EventStruct *event, bool enable, const char *Line)
```

Comment:
Why use the `const char* Line` here?
It is not used, so that's why I did not add it as a function argument.

Reply:
It is needed for this feature to work, transform a TaskName into a TaskIndex.

Reply:
Ah OK.

Reply:
> Can't this be done before calling this function, right where the event is also filled with Var1...Var5

Then we would have to check first if the command is one of the Task* commands, as the `Command_Task_*` functions are called directly from the internal command processing. But this syntax isn't valid/applicable for other commands.